harness/determined/integrations/huggingface/hf_callback.py

```diff
@@ -36,7 +32,9 @@ def __init__(
         self.current_op = next(self.searcher_ops)
         self.updating_searcher = False
 
-        searcher_config = det.get_cluster_info().trial._config["searcher"]
+        cluster_info = det.get_cluster_info()
+        assert cluster_info
```

Comment:
Did `mypy` necessitate the `assert`? Should probably have a `"This example must run on-cluster"` string after the assert, or similar, to let the user know what happened.

Reply:
yeah, basically, mypy understands when you put and assert that it can't be `None` after that. We could add that string in a later PR, I'd like to make sure we are in agreement on the feature branch